lib/card_generator.rb

```diff
@@ -0,0 +1,22 @@
+# require './card'
+
+class CardGenerator
+  attr_reader :card_group
```

Comment:
We don't need this `attr_reader` since `card_group` is not called outside of this class.